hm_pyhelper/miner_param.py

```diff
@@ -60,7 +60,7 @@ def run_gateway_mfr(args):
     except ResourceBusyError as e:
         err_str = "resource busy error: %s"
         LOGGER.exception(err_str % str(e))
-        raise ResourceBusyError(e)\
+        raise ResourceBusyError(e) \
```

Comment:
Good catch.

Reply:
Solved in https://github.com/NebraLtd/hm-pyhelper/commit/7a7bd76fa00312fd28c77fd725d69b070eda83de